Feedback (#1)
sdgis-edu-tud/report-asa2025-groupd: report.qmd

Reviewer: The results of the analysis still to be added in the `Results` section.

---

> title: "Title of your report"

Reviewer: Think of a title and update this in the report.

subtitle: "Project report of Group D"
author:
  - name: "Frederick Auer"
    affiliation: "MSc Geomatics"
    group_role: "Software/Program Helper & Data Collector"
  - name: "Lin Ruofan"
    affiliation: "MSc Landscape Architecture"
    group_role: "Mapping Specialist & Design Lead"
  - name: "Matthijs Jerez Nova"
    affiliation: "MSc Urbanism"
    group_role: "Writer/Editor & Research Lead"
  - name: "Ula Kunigelyte"
    affiliation: "MSc Urbanism"
    group_role: "Mapping Specialist & Data Analyst"
  - name: "Finn van Asch"
    affiliation: "MSc Urbanism"
    group_role: "Research Lead & Assistant Mapping Specialist"

bibliography: references.bib
csl: apa.csl

format: html

editor: visual
---

```{r}
#| label: setup

```

## Introduction

> Urban areas across Europe are increasingly challenged by the impacts of climate change, biodiversity loss, and the demand for sustainable development [@nijkamp2013new]. According to [@article], in cities the impact of land use on fresh-water ecosystems is especially severe. As climate change is expected to increase the intensity and frequency of extreme weather events, solutions to combat these effects are urgently needed [@ourloglou2020assessing].

Reviewer: Placeholder reference to be updated.

A good solution is through urban streams. Urban streams are expected to have an important role in the mitigation of climate change by buffering extreme temperatures and prevent floods. More importantly, finding nature-based solutions are key and can restore existing strained water systems [@ourloglou2020assessing]. In this context, spatial analytics can play a crucial role in informing climate adaptation and urban planning strategies [@hurlimann2012role]. Through methods such as Spatial Multi-Criteria Decision Analysis (S-MCDA) and typology development, planners and designers can better understand urban systems, evaluate trade-offs, and propose the correct targeted interventions.

> The ReBioClim project, a European initiativve funded by the Interreg Central Europe Program [@rebioclim2024], exemplifies this approach by focusing on revitalizing small urban streams. It additionally provides a framework where nature-based solutions can be identified to enhance biodiversity, mitigate climate change effects, and improve urban living conditions [@rebioclim_ioer2024]. Running from Juse 2204 to January 2027, the project addresses the challenges of urban stream restroration in four Central European cities, including Jablonec nad Nisou.

Reviewer: I suggested some changes on this text. Feel free to commit them directly. You can do so with the Commit button here on GitHub directly. 

```suggestion
The ReBioClim project, funded by the Interreg Central Europe Program [@rebioclim2024], exemplifies this approach by focusing on revitalizing small urban streams. It additionally provides a framework where nature-based solutions can be identified to enhance biodiversity, mitigate climate change effects, and improve urban living conditions [@rebioclim_ioer2024]. Running from June 2024 to January 2027, the project addresses the challenges of urban stream restoration in four Central European cities, including Jablonec nad Nisou, the case targeted in this report.
```

![ReBioClim Project Overview](Inserted_images/rebioclim_currentstate.png){width="600"}

> In this report, inspired by the ReBioClim project, we focus on the urban streams of Jablonec nad Nisou. Over recent decades, the city has experienced overbuilding, channelization, and neglect of its urban streams, leading to ecological degradation and reduced quality of life. Our objective is to quantitatively assess the current state of these waterways and identify areas with the highest potential for improvement. Through these efforts, we aim to provide a better understanding of where and how to restore the waterways and create green-blue corridors that offer cooling effects, improve air quality, and provide recreational spaces for residents.

Reviewer: ```suggestion
Over recent decades, Jablonec nad Nisou has experienced overbuilding, channelization, and neglect of its urban streams, leading to ecological degradation and reduced quality of life. Our objective is to quantitatively assess the current state of these waterways and identify areas with the highest potential for improvement. Through these efforts, we aim to provide a better understanding of where and how to restore the waterways and create green-blue corridors that offer cooling effects, improve air quality, and provide recreational spaces for residents.
```

```{r fig.subcap="Jablonec nad Nisou and the Bila Nisa stream"}
#| warning: false
#| echo: false

> #| message: false

Reviewer: @fauer1-bit note that I cannot reproduce the issue you had on your computer. I do not see any messages when I render this code chunk.

library(leaflet)
library(sf)

> # Read the stream GeoJSON without 

Reviewer: ```suggestion
# Read the stream in GeoJSON format
```

stream <- st_read("Inserted_images/stream_for_interactivemap.geojson", quiet = TRUE)

# Coordinates for Jablonec nad Nisou
jablonec_coords <- c(lat = 50.7224, lng = 15.1708)

# Render leaflet map
leaflet() %>%
  addTiles() %>%
  addPolylines(
    data = stream,
    color = "blue",
    weight = 3,
    opacity = 0.8,
    popup = "Bílá Nisa Stream"
  ) %>%
  addCircleMarkers(
    lng = jablonec_coords["lng"],
    lat = jablonec_coords["lat"],
    radius = 6,
    color = "red",
    fillOpacity = 0.8,
    label = "Jablonec nad Nisou"
  ) %>%
  addLegend(
    position = "bottomright",
    colors = c("blue", "red"),
    labels = c("Bílá Nisa Stream", "Jablonec nad Nisou"),
    title = "Legend",
    opacity = 0.8
  )
```

> Our research centers on the Bílá Nisa stream, and will address the problems we have identified through the lens fo the three following goals: Biodiversity, Climate Adaptation, and Quality of Life. These goals, relevant to our location has led to our main research question:

Reviewer: ```suggestion
Our research centers on the Bílá Nisa stream, and will address the problems we have identified through the lens of the three following goals: Biodiversity, Climate Adaptation, and Quality of Life. These goals, relevant to our location have led to our main research question:
```

> *Where should urban stream restoration be prioritized in Jablonec nad Nisou to maximize biodiversity, climate adaptation, and quality of life?*

Our derived subquestions, respective to each goal are then the following:

> *Where can the ecological quality of the stream area be enhanced?*

> > *Which areas are the most in need for flood protection?Where can flood mitigation measures take place?*

Reviewer: ```suggestion
> *Which areas are the most in need for flood protection? Where can flood mitigation measures take place?*
```

> *Where can improvements in quality of life balance stakeholder interests in areas around the stream?*

//we will add how the report is structured here after.

## Methodology

> A important foundation for applying spatial analytics in urban environments is in the careful definition of the area of interest (AOI) and the spatial units used for analysis. The decisions regarding spatial scale, unit configuration, and data resolution directly influences the insights that can be drawn from spatial data. [@dungan2002balanced] The justification of our area of interest and chosen spatial unit is structured around principles derived from literature, geographical context and data availability. Furthermore, in order to apply a S-MCDA normalized values at a comparable scale are needed, further emphasizing the importance of defining an appropriate spatial unit.

Reviewer: Which sentence does `[@dungan2002balanced]` refer to?

> **Definition of Spatial Units**

Reviewer: I suggest you use markdown headings (`###`-level should be okay here) instead of **bold**-ed phrases

> **Definition of Spatial Units**

Reviewer: ```suggestion
### Definition of Spatial Units
```

Reviewer: I made a suggestion here, apply to all similar occurrences in the report.

To conduct a meaningful spatial analysis, the study area must be divided into standardized spatial units. For example, urban heat island and green infrastructure studies have shown to use a regular grid of 50 x 50 meter cells as the base unit of analysis [@BARTESAGHIKOC2020103893]. On the other hand, [@burdziej2019using] uses a hexagonal grid where each hexagon has a side length of 100 m, selected based on the size of the study area (Toruń, Poland) and the intended spatial resolution of the study.

Therefore, based on the size of Jablonec nad Nisou and the Bila Nisa stream as well as the following discussed principles affecting the conducted analysis, a spatial unit grid of 100 x 100 meters was chosen. The unit is used for consistency in the assignment of spatial indicators and facilitates the aggregation and comparison of data across the urban stream landscape. The 100 x 100m spatial unit is specific to the scale Jablonec nad Nisou, the resolution of the data collected and relevant literature on stream assessments.

**Resolution and Data Acquisition Method**

> The resolution of spatial analysis is directly tied to the data acquisition method [@BARTESAGHIKOC2020103893]. While spaceborne remote sensing, used for the measurement of some of our indicators when data availability was lacking, is better suited for large-scale urban assessments and often lacks the spatial precision needed for detailed micro climatic or ecological studies. On the other hand biodiversity and quality of life data are generally acquired at larger scale and would require smaller spatial units. For example, the widely accepted walking range of 500 meters [@macioszek2022effect; @popp2004walking], or biodiversity connectivity of around 100m stretches, identified by [@RANTA2021106980]. Our method reflects this balance, using satellite data for broader patterns such as LST and impervious surfaces. We then supplement this with higher-resolution sources for space syntax calculations, biotope values and ecologically valuable habitats, to name a few. The collected data for each respective goals can be found in the figures below.

Reviewer: ```suggestion
The resolution of spatial analysis is directly tied to the data acquisition method [@BARTESAGHIKOC2020103893]. While spaceborne remote sensing used for the measurement of some of our indicators when data availability was lacking is better suited for large-scale urban assessments, it often lacks the spatial precision needed for detailed micro climatic or ecological studies. On the other hand, biodiversity and quality of life data are generally acquired at larger scale and would require smaller spatial units. For example, the widely accepted walking range of 500 meters [@macioszek2022effect; @popp2004walking], or biodiversity connectivity of around 100m stretches, identified by [@RANTA2021106980]. Our method reflects this balance, using satellite data for broader patterns such as LST and impervious surfaces. We then supplement this with higher-resolution sources for space syntax calculations, biotope values and ecologically valuable habitats, to name a few. The collected data for each respective goals can be found in the figures below.
```
